cinema/views.py

```diff
@@ -43,11 +62,50 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset.prefetch_related("actors", "genres")
+
+        actors = self.request.query_params.get("actors")
+        genres = self.request.query_params.get("genres")
+        title = self.request.query_params.get("title")
+
+        if actors:
+            actors_ids = [int(str_id) for str_id in actors.split(",")]
```

Comment:
The better way is to create an additional method for separating one string into a few str_ids and not duplicate code in lines 73 and 77.